eclipse-glsp/glsp-theia-integration
# GLSP-1320: Update to latest glsp dev config

#### What it does

- Updates to latest version of dev,client & server packages
- Add temporary resolution 
- Add generate:index utility script
- Regenerate index files
- Fix codeActionsOnSave in vscode settings
- Refactor upgrade:next script
- Reformat code base with prettier
- Fix copyright headers (by running a full check with glsp checkHeaders . -t full)

Requires: https://github.com/eclipse-glsp/glsp-client/pull/345
Part of eclipse-glsp/glsp#1320

#### How to test



#### Follow-ups



#### Changelog
